usr/lib/pkcs11/ep11_stdll/ep11_specific.c

```diff
@@ -394,6 +395,20 @@ typedef struct const_info {
 #define CKM_IBM_WIRETEST                   CKM_VENDOR_DEFINED + 0x00030004
 #define CKM_IBM_RETAINKEY                  CKM_VENDOR_DEFINED + 0x00040001
 
+static CK_RV cleanse_key_attribute(TEMPLATE *template,
```

Comment:
shouldnt this be called cleanse_attribute ? i mean it would work for all attributes .

Reply:
fixed